### ISSUE TITLE
Choose which Microphone on Microphone array 

### ISSUE THREAD

**Issue by akramhussein** [respeaker/respeaker_python_library#32]
Hi - is it possible to select which Microphone is activated or to disable some? I'd like to manually be able to control which Mic is on.

**Comment by KillingJacky** [comment]
Sorry we can't control the microphone, but you can drop the data of specific channel(s) when you using the captured data later.

**Comment by akramhussein** [comment]
Hi @KillingJacky - thanks for the reply.

I've tried using the 8-channel firmware but then I have to run a lot of signal processing algorithms at a high level and I'd like to avoid this as its computationally heavy and it's what the XMOS was designed for and I'd like to keep all the Beamforming, Noise Suppression, De-reverberation and Acoustic Echo Cancellation

So to clarify, there is no way to to suggest to the XMOS "please focus on this direction", rather than let it determine where to listen? You must either accept it choosing the loudest person talking or get all 8 channels and do it yourself?

Thanks!

**Comment by KillingJacky** [comment]
Hi @akramhussein 
Thanks for the clarification, now I get your concern. But here's still an unlucky story :( We've tried to do what you wanna do as well, all we have is a HID register list - [here](https://github.com/respeaker/respeaker-xmos-hid/blob/master/registers.csv), we've tried several combinations of register value for REG0x16 and REG0x17, but the beam still not gonna focus the direction we specified. And XMOS can not give solutions too. I think the registers there are dummy, and the beamforming algorithm inside it can not be configured, but only outputs the angle via REG0x45.
